lib/file_validators/validators/file_content_type_validator.rb

```diff
@@ -9,16 +9,20 @@ def self.helper_method_name
       end
 
       def validate_each(record, attribute, value)
-        value = JSON.parse(value) if value.is_a?(String) && value.present?
-        unless value.blank?
-          mode = option_value(record, :mode)
-          content_type = get_content_type(value, mode)
-          allowed_types = option_content_types(record, :allow)
-          forbidden_types = option_content_types(record, :exclude)
-
-          validate_whitelist(record, attribute, content_type, allowed_types)
-          validate_blacklist(record, attribute, content_type, forbidden_types)
-        end
+        values = parse_values(value)
+        mode = option_value(record, :mode)
+        allowed_types = option_content_types(record, :allow)
+        forbidden_types = option_content_types(record, :exclude)
+
+        values.all? do |v|
+          content_type = get_content_type(v, mode)
+          valid_whitelist?(record, attribute, content_type, allowed_types)
+        end or mark_invalid record, attribute, :allowed_file_content_types, allowed_types
```

Comment:
It would be better, if we could avoid this syntax. It makes the code almost unreadable.